libdnf/comps/comps.cpp

```diff
@@ -0,0 +1,75 @@
+#include <libdnf/comps/comps.hpp>
+#include <libdnf/comps/comps_impl.hpp>
+
+extern "C" {
+#include <solv/pool.h>
+#include <solv/repo.h>
+#include <solv/repo_comps.h>
+}
+
+#include <filesystem>
+#include <iostream>
+
+namespace libdnf::comps {
+
+
+Comps::Comps(libdnf::Base & base) : base{base}, p_impl{new Impl()} {}
+
+
+Comps::~Comps() {}
+
+
+void Comps::load_installed() {
+    // TODO(pkratoch): load real installed groups
+    std::filesystem::path data_path = PROJECT_SOURCE_DIR "/test/libdnf/comps/data/";
+    std::string mock_path = data_path / "core.xml";
+
+    Pool * pool = p_impl->get_pool();
+    Repo * repo = 0;
+    Id repoid;
+    // Search for repo named @System
+    FOR_REPOS(repoid, repo) {
+        if (!strcasecmp(repo->name, "@System")) {
```

Comment:
Note: it would be better to use  `SYSTEM_REPO_NAME` from solv_sack.cpp, but that's not a public header. No action needed now.